algorithms/src/snark/marlin/data_structures/circuit_proving_key.rs

```diff
@@ -52,9 +52,9 @@ impl<E: PairingEngine, MM: MarlinMode> ToBytes for CircuitProvingKey<E, MM> {
 impl<E: PairingEngine, MM: MarlinMode> FromBytes for CircuitProvingKey<E, MM> {
     #[inline]
     fn read_le<R: Read>(mut reader: R) -> io::Result<Self> {
-        let circuit_verifying_key = CanonicalDeserialize::deserialize(&mut reader)?;
-        let circuit_commitment_randomness = CanonicalDeserialize::deserialize(&mut reader)?;
-        let circuit = CanonicalDeserialize::deserialize(&mut reader)?;
+        let circuit_verifying_key = CanonicalDeserialize::deserialize_compressed(&mut reader)?;
+        let circuit_commitment_randomness = CanonicalDeserialize::deserialize_compressed(&mut reader)?;
+        let circuit = CanonicalDeserialize::deserialize_compressed(&mut reader)?;
```

Comment:
Sanity checking: were these previously serialized in compressed or uncompressed form?

Reply:
old `deserialize` = new `deserialize_compressed`, so this doesn't change any behavior. (This can also be seen from the fact that no tests are failing